doc/manual/rule_uid_schema.md

```diff
@@ -150,24 +150,24 @@ match `asam.net:xodr:1.8.0:road.planview.geometry.ref_line_exists`.
 
 There are different alternatives to define a rule full name. Those alternatives
 are domain specific and this document does not restrict the strategies applied
-to define them. 
+to define them.
 
 The format of the full name uses a well defined notation, compatible with the
 following pattern:
 
-```
+```text
```

Comment:
I suggest to disable this automatic insert `text` on code block. This code block is actually `pcre` but the support is not standard markdown (github does not support it) but many other readers do. Inserting `text` forces a wrong detection (and `text` is also default fallback, does not need to be explicit).

Reply:
I've added them manually. Reason was because of markdownlint didn't like it https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md040.md and proposes `text` for plain text in code block.

Reply:
Added `pcre` as language identifier as proposed. Github fallback on to standard `text` works perfect.